cmd/mt-index-cat/main.go

```diff
@@ -251,13 +252,13 @@ func main() {
 	for _, d := range defs {
 		// note that prefix and substr can be "", meaning filter disabled.
 		// the conditions handle this fine as well.
-		if !strings.HasPrefix(d.Name, prefix) {
+		if !strings.HasPrefix(d.Name.String(), prefix) {
```

Comment:
out of scope for v1, but in the future we may add custom HasPrefix/HasSuffix/Contains functions that don't require decoding the entire string up front